contrib/completion/bash/docker

```diff
@@ -70,6 +85,46 @@ __docker_container_ids() {
 	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
 }
 
+__docker_images() {
+	local images_args=""
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_IDS" in
+		all)
+			images_args="--no-trunc -a"
+			;;
+		non-intermediate)
+			images_args="--no-trunc"
+			;;
+	esac
+
+	local awk_script=''
+	case "$DOCKER_COMPLETION_SHOW_IMAGE_IDS" in
+		all|non-intermediate)
+			case "$DOCKER_COMPLETION_SHOW_TAGS" in
+				yes)
+					awk_script='NR>1 { print $3; if ($1 != "<none>") { print $1; print $1":"$2 } }'
```

Comment:
Thanks for the update, Rory. I found no problems any more.

But I suggest one more refactoring: If you pull out the part that prints the repo names (`print $1; print $1":"$2` vs `print $1`) from `awk_script` into a new variable and insert its value here, you get rid of some repitition and reduce nesting. What do you think?